core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/inventory/InventoryTriggerLoader.java

```diff
@@ -84,8 +84,10 @@ public void save(InventoryTrigger trigger) {
 
             IItemStack[] items = trigger.getItems();
             int size = items.length;
+            boolean pickup = trigger.isPickup();
```

Comment:
Same with the above thing.
```suggestion
            boolean pickup = trigger.isPickup();
            boolean canPickup = trigger.canPickup();
```